src/server/mod.rs

```diff
@@ -81,7 +92,7 @@ impl<P: Publisher> Server<P> {
     #[instrument(level = "info", skip_all)]
     pub async fn serve(mut self) -> anyhow::Result<()> {
         while let Some(msg) = self.subscriber.next().await {
-            if !msg.subject.starts_with(&self.prefix) {
+            if !msg.subject.starts_with(&self.prefix) && !self.multitenant {
                 warn!(subject = %msg.subject, "Received message on an invalid subject");
```

Comment:
When would this happen anyway? I assume the subscription was always on `{prefix}.>` before, and now can also be `*.{prefix}.>`